app/assets/stylesheets/application.css.scss

```diff
@@ -91,6 +91,28 @@ input {
   padding-top: 12%;
 }
 
+.jumbotron > #form-c {
+  background: none !important;
+  text-align: center;
+}
+
+.jumbotron > form > input[type="text"] {
+ border-radius: 4px;
+ border: 0px;
+ width: 30%;
+ margin-top: 1px;
+ height: 30px;
+ text-align: center;
+}
+
+#form-ul > form > input[type="text"] {
+ border-radius: 4px;
+ border: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>`0px` should be written without units as `0`